### ISSUE TITLE
Accessibility: On Windows, on the interstitial page, the price boxes and the “Limited-time only” message are not read/picked up by a browser screen reader

### ISSUE THREAD

**Issue by AOiegas** [mozilla/fx-private-relay#1480]
#### Build:
- Stage;

#### Affected Platforms:
- Windows 10;

#### Browsers tested on:
- Firefox, latest version;

#### Prerequisites:
- Be in a country, or have a VPN set to a country which has Premium available;
- Do not be signed in;
- Have a browser screen reader/text to speech application installed (e.g. Pericles);

#### Steps to reproduce:
1. Go to the Stage Interstitial Page;
2. Scroll down to the bottom of the page;
3. Use the screen reader to read the price plan boxes;

#### Expected result:
- The screen reader reads the content of the boxes without issue;

#### Actual result:
- The screen reader doesn’t read/pick up the content of the boxes;

#### Notes:
- I used Pericles as it seems to be popular and it was also suggested by a person that has more experience with screen readers;
- All my tries with Pericles stopped reading at the Use Cases (Shopping, Social Networks, Gaming) and did not continue below to the price plan or the "Limited-time only" text;
- See attached media:
![image](https://user-images.githubusercontent.com/85748693/150100684-6549a09b-0e98-428c-8119-c81b4aa47ba1.png)
